services/bots/helpers.py

```diff
@@ -0,0 +1,32 @@
+from shared.config import get_config
```

Comment:
It took me a while to realize that `get_config` was actually loading from YAML configurations. `from shared.config import get_config` just looked a lot like it's loading application configs, not YAML (I had to dig into the code to verify that it's actually YAML). Anyways, not sure if it's a _me_ thing, or if other people will also be confused.

Reply:
> from shared.config import get_config just looked a lot like it's loading application configs

That is the case. The fact that the config lives in a YAML is just an implementation detail. For all intents and purposes it's fine to look at `get_config` as "getting application configs" and understand that the config logically is a map